uniffi_meta/src/lib.rs

```diff
@@ -272,7 +272,7 @@ pub enum LiteralMetadata {
     EmptySequence,
     EmptyMap,
     None,
-    Some { inner: Box<LiteralMetadata> },
+    Some { inner: Box<DefaultValueMetadata> },
```

Comment:
I think this might be the issue, this value should still be a `LiteralMetadata` right?